### ISSUE TITLE
Unable to use link with paranthesis on the editor

### ISSUE THREAD

**Issue by lidorcg** [GitbookIO/feedback#203]
### What is the current behavior?

Hello I'm using the gitbook editor and there's seems to be a problem with some links:
as I looked I saw that your using the [words](link) syntax for links, however, when I'm trying to insert links that has parenthesis in them (mostly form github and wikipedia) the first closing parenthesis in the link itself seems to close the link parenthesis of the syntax which leaves me with a broken link and visual awkwardness in the page, here's an example:

`##### [Om Next](https://github.com/omcljs/om/wiki/Quick-Start-(om.next)\)`

which renders into something like this:
##### [Om Next](https://github.com/omcljs/om/wiki/Quick-Start-%28om.next%29))

only with a link to:
https://github.com/omcljs/om/wiki/Quick-Start-(om.next
### What is the expected behavior?

Well I think this bug is pretty self explanatory and it seems like a really bad bug as I couldn't even reproduce it in the md complier of github. 
I should specify that I tried to use a different link syntax but as I save, the editor seems to rearrange the code and bring back this code up here...
#### For GitBook Editor
- Operating System?
  Ubuntu 16.04 LTS
- (Web) Browser version?
  Mozilla firefox 49.0 I think...
- (Desktop) Editor version?
  6.2.1
- Was the bug present in previous versions? 
  I think it's my first version so I can't be sure...

**Comment by nagim** [comment]
Hi,

I'm pretty sure that the backslash (\) at the end of the link is causing the problems and it's unnecessary to have it there. It usually used to escape syntax elements and render them literally. Try to remove it and test again.

Cheers,
Imre

**Comment by lidorcg** [comment]
Hi Imre, thanks for replying

Well actually there is nothing wrong with the md cause, as I found out
recently,  in the web editor it renders just fine.
Moreover, I didn't write the md, it is automatically generated by the
gitbook desktop editor,
I've tried numerous ways to write this link but the gitbook code generator
keeps on rewriting it like this.

Since, It's working just fine on the web site it's no biggie, but it is a
bug on the gitbook desktop app md renderer (I think..).

Hoped I helped,
Lidor

2016-10-12 9:21 GMT+03:00 nagim <notifications@github.com>:
